### PR TITLE
Aboutme section complete

### PR DESCRIPTION
**Pull request by ometman**
I have completed the about me section. A brief description of myself, and my social media handles were added, and languages, frameworks and skills were also added as links..
I have updated the about-me section, aligning sides, including button behaviors and skills section color change. Styles modified accordingly

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -317,5 +323,333 @@
...
+    margin: 0 48px 0 12px;
+    height: 222px;
+    width: 327px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-items: center;
+    gap: 12px;
+    float: none;
+  }
```
Comment (Tufahel) [inline css/styles.css:605]: - [ ] As per the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) there should be some extra `space` between the border and the `javascript`, `html` and `css` icon. Kindly fix this to match the design below.
  - ***Figma Design***
  Screenshot from 2022-12-10 00-00-02
  - ***Your Design***
  Screenshot from 2022-12-10 00-01-13

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ometman, :wave: 

Good Work So Far!!! :+1:  
You have implemented some required changes from the previous reviewer. There are still some issues you need to fix.

## Highlights 🌟

Alt Text
✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (ometman) [reply css/styles.css:605]: All the comments on the review have been observed for implementation.

File: index.html [hunk 2]
```diff
@@ -60,5 +60,229 @@ <h1 class="primary-text">
...
+                            
+                            
+                                <li>
+                                <a href="#"><img class="item-icon" src="./images/icons/button-icon.png" alt="skill selector button"></a> 
+                                </li>
+                            </a> 
+                        </ul>
+                        
+                    </div>
+                </div>
```
Comment (Qoosim) [inline index.html:284]: - [ ] As mentioned by the previous reviewer, you're advised to stick to the figma design as close as possible. Kindly look into the background color of your design and that of the figma design. The background color of your design is a bit darker to what is on the figma design. Also left padding on the each container of the language has not been implemented. Kindly attend to that too.

### Your Design
your-design

### Figma Design
figma-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @ometman, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ometman) [reply index.html]: Effected alignment changes at skills section of about me section.  I used the original colors at found but the rendering seems a little different as per system. Thanks

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ometman, 

Your project is complete! Great job for following the project requirement🥇and implementing changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations! 🎉💯🌟 



## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

- [ ] [OPTIONAL but recommended] I would kindly suggest you stick to the Figma template design as much as possible. I notice some light grey spaces outside your design responsiveness. which implies that your project is not fitting into the `375px` screen size as requested in the project requirement. Kindly find a way to fix that.

Your design 👇 

omet


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — COMMENTED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @ometman ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**